src/features/machine/PolarSettings.js

```diff
@@ -61,14 +65,37 @@ class PolarSettings extends Component {
               </Col>
             </Row>
 
-            <Accordion>
-              <Card className={`${endpointsActiveClass} overflow-auto`}>
-                <Accordion.Toggle as={Card.Header} eventKey={0} onClick={this.props.toggleEndpoints}>
-                  <h3>Force endpoints</h3>
-                  Forces the first and last points to be at the center and edge
-                </Accordion.Toggle>
-              </Card>
-            </Accordion>
+            <Row className="align-items-center pb-2">
+              <Col sm={4}>
+                <Form.Label htmlFor="forceStart">
+                  Start point
+                </Form.Label>
+              </Col>
+
+              <Col sm={8}>
+                <ToggleButtonGroup id="forceStart" type="radio" name="origin" value={this.props.startPoint} onChange={this.props.onStartPointChange}>
+                  <ToggleButton variant="light" value={0}>auto</ToggleButton>
```

Comment:
I think "auto" may not be the best world. Maybe "none" instead?

Reply:
The concept of auto is actually what I was hoping for. Something where the start and end points would would be determined by how far away the pattern's start and end was from the center.

Reply:
We could also make them toggle, so if you had auto it would pick, but if you selected none of them, it wouldn't put the start or end in.

Reply:
I think string (enum) is a good approach. And I'm not sure auto is what you want in the context of a playlist - the user needs to decide. I'll make the changes soon.